app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = []
+    for person in people:
+        new_person = Person(person["name"], person["age"])
+
+        married_key = "wife" if "wife" in person else "husband"
+        married_name = person.get(married_key)
+        married = Person.people.get(married_name)
```

Comment:
Try to rename this variables, the names do not correspond to what is in them

Reply:
How I should rename variables? We are talking about married_key...?
Is "spouse" instead "married" okay?

Reply:
Yes, it is great!